front-page.php

```diff
@@ -30,20 +30,21 @@
     $subtitle = get_post_meta( $post->ID, '_igv_post_subtitle', true );
   ?>
 
-      <article <?php post_class(); ?> id="featured-post-container">
-        <div id="featured-post-title-holder">
-          <h3 id="featured-post-title" class="rotate-text js-fix-widows">
+      <article <?php post_class('row'); ?> id="featured-post-container">
+        <div id="featured-post-title-holder" class="col s2">
+          <h3 id="featured-post-title" class="rotate-text">
```

Comment:
the line height for both the <h3> and <h4> are wrong. The h3 needs less and the h4 more. Possibly the h3 is a little too big now. I can do this by eye once everything else is good